main.py

```diff
@@ -85,7 +85,7 @@ def main() -> None:
     )
     output_file_name = parser.parse_args().output_file_name or "current_fees.csv"
     fees_file_name = parser.parse_args().fees_file_name or "current_fees_collected.json"
-    fees_path = os.path.join(ROOT, "fee_allocator", "fees_collected", fees_file_name)
+    fees_path = f"fee_allocator/fees_collected/{fees_file_name}"
```

Comment:
why this change? `os.path.join` works better in some cases (mostly different file systems)

Reply:
It wasn't working.  I changed it to make it find the file.   Somehow the python version upgrade broke it.  You're welcome to try if you can figure it out.

Reply:
ah ok well in that case changing it is a good reason :)